src/django/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN set -ex \
     && buildDeps=" \
     build-essential \
     " \
-    && apt-get update && apt-get install -y $buildDeps --no-install-recommends \
+    && apt-get update && apt-get install -y git $buildDeps --no-install-recommends \
```

Comment:
Revert prior to merge. This is only necessary to install `django-ecsmanage` via Git.